src/functions/carts/getCart/handler.ts

```diff
@@ -2,12 +2,31 @@ import 'source-map-support/register';
 
 import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/apiGateway';
 import { middyfy } from '@libs/lambda';
-import Cart from '@dbModel/tables/cart';
-import Response from '@lamdaModel/lambdaResponse';
+import CartRow from '@dbModel/tables/cart';
+import Response from '@lamdaModel/lambdaResponse'
+import dbContext from '@dbModel/dbContext';
+import HttpStatusCodes from '@lamdaModel/httpStatusCodes';
 
-const scan: ValidatedEventAPIGatewayProxyEvent<any> = async (event) => {
-    let response: Response<Cart>;
+
+/*
+ * Remember: event.body type is the type of the instantiation of ValidatedEventAPIGatewayProxyEvent
+ * In this case event.body type is 'void' -> we have no body!
+*/
+const getCart: ValidatedEventAPIGatewayProxyEvent<void> = async (event) => {
+    let response: Response<CartRow[]>;
+    let cart: CartRow[] = new Array<CartRow>();
+    try {
+        for await (const element of dbContext.scan(CartRow)) {
```

Comment:
Pay attention to pagination, see [https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Scan.html](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Scan.html), section 'Paginating the Results'.
I suggest to use array `concat` method to store the page.
For example see the the function scanProduct.